src/testRunner/unittests/shimMap.ts

```diff
@@ -1,5 +1,5 @@
 namespace ts {
-    describe("unittests:: shimMap", () => {
+    describe("unittests:: createMapShim", () => {

```

Comment:
This makes the suite name not contain the file bame, which, unfortunately breaks my normal f5-to-test-active-file launch.json - could ya swap it back or rename the file?